SA3/qa_web/templates/search/indexes/qa_web/Questions_text.txt

```diff
@@ -0,0 +1,6 @@
+{{ object.title }}
```

Comment:
After further testing, this file's name with a capital letter makes it unable to be found by Haystack. it should be renamed to (case sensitive) `questions_text.txt`

Reply:
I have tested this thought and it's not the root cause. The instruction of Haystack says the name should be ModelName_text.txt so i used Questions at first place.

Reply:
Either you change this file name to questions_text.txt **or you look at my next comment and you change the template name in `line 8` of `search_indexes.py`**. If not, Django will raise an error, as seen below. 

![image](https://user-images.githubusercontent.com/29107916/37689308-adecb33a-2c7a-11e8-80e3-86c0ca385537.png)

Right now, I cannot test your feature because this error is blocking `python manage.py rebuild_index`

Reply:
To make sure we keep a consistent naming pattern for templates, the file should be named following under_score practices: `question_text.txt`.